### PR TITLE
Localization for waste types

### PR DESCRIPTION
**Pull request by NikPeters-Atos**

### CODE REVIEW

File: Portal/Helpers/TagHelpers/RadioTagHelper.cs [hunk 1]
```diff
@@ -42,17 +42,17 @@ public override async Task ProcessAsync(TagHelperContext context, TagHelperOutpu
 
             var dataProperty = AspFor;
 
-            if (AspDataFor != null)
-                dataProperty = AspDataFor;
+//            if (AspDataFor != null)
```
Comment (shehzad-ismail8) [inline Portal/Helpers/TagHelpers/RadioTagHelper.cs]: Do we want to keep these commented out lines?

**Review by shehzad-ismail8 — COMMENTED** [review]
One comment about commented out code

File: Portal/Helpers/TagHelpers/RadioTagHelper.cs (hunk 1 above)
Comment (NikPeters-Atos) [reply Portal/Helpers/TagHelpers/RadioTagHelper.cs]: Good spot. Done

**Review by shehzad-ismail8 — APPROVED** [review]
Looks good to me

**Pull request merged**